simulator/opcuatms.service

```diff
@@ -1,14 +0,0 @@
-<?xml version="1.0" standalone='no'?><!--*-nxml-*-->
-<!DOCTYPE service-group SYSTEM "avahi-service.dtd">
-<service-group>
-  <name replace-wildcards="yes">%h</name>
-  <service>
-    <type>_opcua-tcp._tcp</type>
-    <port>4840</port>
-    <txt-record>name=Reference device simulator</txt-record>
-    <txt-record>manufacturer=openDAQ</txt-record>
-    <txt-record>model=Reference device</txt-record>
-    <txt-record>serialNumber=dev_ser_1</txt-record>
-    <txt-record>caps=OPENDAQ</txt-record>
-  </service>
-</service-group>
```

Comment:
I'm not 100% certain we want to completely remove the avahi services. We could leave OPC UA using avahi, and native using our internal discovery service.